rebasehelper/tests/test_utils.py

```diff
@@ -42,6 +44,32 @@
 from rebasehelper.utils import LookasideCacheHelper
 
 
+class TestGitHelper(object):
+
+    @pytest.yield_fixture
+    def temp_dir(self):
+        current_dir = os.getcwd()
+        try:
+            with TemporaryEnvironment() as temp:
+                temp_repo_path = temp.path()
+                os.chdir(temp_repo_path)
+                yield
+        finally:
+            os.chdir(current_dir)
+
+    @pytest.mark.usefixtures('temp_dir')
```

Comment:
You can use existing [`workdir` fixture](https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/tests/conftest.py#L33-L40).

Reply:
ok I will use it.